js/Ball.js

```diff
@@ -5,14 +5,45 @@ class Ball {
         this.ballRadius = 10; // promień piłeczki
     }
 
-    drawBall(){
+    drawBall(ballX, ballY){
         // TO DO -> metoda tworząca piłeczkę (metody canvas'a)
-        // hint: jako argument funkcja przyjmuję x i y zdefiniowane globalnie w index.js 
+        // hint: jako argument funkcja przyjmuję x i y zdefiniowane globalnie w index.js
+        this.ctx.beginPath();
+        this.ctx.arc(ballX,ballY,this.ballRadius,0,2*Math.PI);
+        this.ctx.stroke();
     };
 
-    bouncing(){
+    bouncing(ballX, ballY, paddle){
         // TO DO -> funcja odbijąca piłeczkę od ścian
         // hint: jako argumenty funkcja przyjmuje x i y zdefiniowane globalnie w index.js oraz pozycję i szerokość paddle
         // hint: gdy piłeczka wpadnie w dolną ścianę -> game over, czyli alert oraz przeładowanie dokumentu i wyczesczenie intarwału
+        
+        // Od prawej i lewej ściany
+        if (ballX <= this.ballRadius || this.cvs.width-ballX <= this.ballRadius){
+            dx = -dx;
+        }
+        // Od górnej ściany
```

Comment:
Trochę sobie skomplikowałeś to odbijanie w pionie, a z tą paletką to już wgl :D 

 if (ballY + dy < this.ballRadius) {
            dy = -dy;
        } else if (ballY + dy > this.cvs.height - this.ballRadius) {
            if (ballX > paddleX && x < paddleX + paddleWidth) {
                dy = -dy;
            } else {
                alert((`Game over! Your score: ${score}`);;
                window.clearInterval(interval);
                location.reload(true); 
            }
        }  
    };

Reply:
Zmienić to?
Pomyślałem, że mam zrobić odbijanie od paletki, a nie ściany w miejscu paletki.